apps/src/templates/EnhancedSafeMarkdown.jsx

```diff
@@ -48,6 +48,58 @@ export const ExpandableImagesWrapper = connect(null, dispatch => ({
   },
 }))(UnconnectedExpandableImagesWrapper);
 
+// A callout can be added via markdown instructions like this:
+//   [play](#showcallout=play-sound-block)
+//
+// This code will replace any link that has #showcallout=[id]
+// with bold clickable text that, when clicked, will call the
+// provided showCallout function with the ID.
+export class ShowCalloutsWrapper extends React.Component {
+  static propTypes = {
+    showCallout: PropTypes.func.isRequired,
+    children: PropTypes.oneOfType([
+      PropTypes.arrayOf(PropTypes.node),
+      PropTypes.node,
+    ]).isRequired,
+  };
+
+  componentDidMount() {
+    this.postRenderHook();
+  }
+
+  componentDidUpdate(prevProps) {
+    // TODO: do we need to do any kind of cleanup here? Or otherwise do
+    // something more precise than calling the method again when we're
+    // responding to an update rather than an initial render?
+    this.postRenderHook();
+  }
+
+  postRenderHook() {
+    const thisNode = ReactDOM.findDOMNode(this);
+    this.renderShowCallouts(thisNode);
+  }
+
+  renderShowCallouts(node) {
+    const showCallouts = node.querySelectorAll('a[href]');
+    for (const showCallout of showCallouts) {
+      const regexp = /#showcallout=(.*)/;
+      const matches = showCallout.href.match(regexp);
+      if (matches?.length === 2) {
+        const calloutId = matches[1];
+        const textContent = showCallout.childNodes[0].textContent;
+        var newNode = document.createElement('span');
```

Comment:
should be `const`. Don't we have a linter rule that prevents `var` declarations?

Reply:
Fixed.  And.. apparently not?